Slambook/index.js

```diff
@@ -1,18 +1,18 @@
-let form;
-let allSlamArray=[];
-//Write code below to select the form element
```

Comment:
Consider using more descriptive comments to explain the purpose of selecting the form element